websites/N/Nebula/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "189690228292845568",
+		"name": "angelolz"
+	},
+	"service": "Nebula",
+	"description": {
+		"en": "Nebula is the home of smart, thoughtful videos, podcasts, and classes from your favorite creators."
+	},
+	"url": "nebula.tv",
+	"matches": [
+		"*://nebula.tv/*"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/VnwSEyu.png",
+	"thumbnail": "https://i.imgur.com/fNXHoQg.jpeg",
```

Comment:
I'm not sure why, but I get 404 for this. 
However, the previous one (`https://i.imgur.com/4wI2vwl.jpeg`) seems to work now. Perhaps its a caching/dns problem on my side.